chapters/publication.tex

```diff
@@ -1,96 +1,178 @@
+
+Dynamic documents are a broad class of tools that enable a streamlined, reproducible workflow.
+The term ``dynamic'' can refer to any document-creation technology
+that allows the inclusion of explicitly encoded linkages to raw output files.
+This means that, whenever outputs are updated,
+the next time the document is loaded or compiled, it will automatically include
+all changes made to all outputs without any additional intervention from the user.
+This means that updates will never be accidentally excluded,
+and it further means that updating results will not become more difficult
+as the number of inputs grows,
+because they are all managed by a single integrated process.
+
+You will note that this is not possible in tools like Microsoft Office,
+although there are various tools and add-ons that produce similar functionality,
+and we will introduce some later in this book.
+In Word, by default, you have to copy and paste each object individually
+whenever tables, graphs, or other inputs have to be updated.
+This creates complex inefficiency: updates may be accidentally excluded
+and ensuring they are not will become more difficult as the document grows.
+As time goes on, it therefore becomes more and more likely
+that a mistake will be made or something will be missed.
+Furthermore, it is very hard to simultaneously edit or track changes
+in a Microsoft Word document.
+It is usually the case that a file needs to be passed back and forth
+and the order of contributions strictly controlled
+so that time-consuming resolutions of differences can be avoided.
+Therefore this is a broadly unsuitable way to prepare technical documents.
+
+There are a number of tools that can be used for dynamic documents.
+In the first group are code-based tools such as R's RMarkdown\sidenote{\url{https://rmarkdown.rstudio.com/}}
+and Stata's \texttt{dyndoc}\sidenote{\url{https://www.stata.com/manuals/rptdyndoc.pdf}}.
+These tools ``knit'' or ``weave'' text and code together,
+and are programmed to insert code outputs in pre-specified locations.
+Documents called ``notebooks'' (such as Jupyter\sidenote{\url{https://jupyter.org/}}) work similarly,
+as they also use the underlying analytical software to create the document.
+These types of dynamic documents are usually appropriate for short or informal materials
+because they tend to offer limited editability outside the base software
+and often have limited abilities to incorporate precision formatting.
+
+The second group of dynamic document tools do not require
+direct operation of underlying code or software, but simply require
+that the writer have access to the updated outputs.
+One very simple one is Dropbox Paper, a free online writing tool
+that allows linkages to files in Dropbox,
+which are then automatically updated anytime the file is replaced.
+Dropbox Paper has very limited formatting options,
+but it is appropriate for working with collaborators who are not using statistical software.
```

Comment:
Suggest the following to reword this paragraph: 
There are other dynamic document tools which do not require direct operation of the underlying code or software, simply access to the updated outputs. These can be useful for working on informal outputs, such as blogposts, with  collaborators who do not code. An example of this is Dropbox Paper, a free online writing tool that allows linkages to files in Dropbox, which are automatically updated anytime the file is replaced.